.github/workflows/generate-weekly-report.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
         with:
           repository: gentoo/gentoo
           path: gentoo
-          ref: master
```

Comment:
This needs to stay as master - it's https://github.com/gentoo/gentoo repo, and main branch there is called master.

Reply:
got a bit carried away here lol, should be fixed now. Thanks for spotting that